index.html

```diff
@@ -7,104 +7,132 @@
-            standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-        <ul class="container-list">
-            <li>css</li>
-            <li>html</li>
-            <li>bootstrap</li>
-            <li>Ruby</li>
-        </ul>
-        <button class="see-project-btn">See Project</button>
+        <div class="my-recent-work">
+            <div style="flex-grow: 1;"></div>
+            <div class="recent-work-content">
+                <h2>My Recent Work</h2>
+                <hr>
+            </div>
+            <div style="flex-grow: 1;"></div>
+        </div>
+        <section class="multi-post-stories">
+            <img src="images/Img Placeholder.png" alt="">
+            <div>
+                <h1>Multi-Post Stories</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry's
+                    standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="container-list">
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="see-project-btn">See Project</button>
+            </div>
+        </section>
     </section>
     <section class="project-container">
-        <div class="project">
-            <h2>Profesional Art Printing Data</h2>
+        <div class="project first-project">
+            <button>See Project</button>
+        </div>
+        <div class="project card2">
+            <h2>Data Dashboard Healthcare</h2>
             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
                 industry's standard</p>
             <ul>
                 <li>html</li>
                 <li>bootstrap</li>
                 <li>Ruby</li>
             </ul>
-            <button>See Project</button>
         </div>
-    </section>
-    <section class="project-container">
-        <div class="project">
-            <h2>Profesional Art Printing Data</h2>
+        <div class="project card3">
+            <h2>Website Protfolio</h2>
             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
                 industry's standard</p>
             <ul>
                 <li>html</li>
                 <li>bootstrap</li>
                 <li>Ruby</li>
             </ul>
-            <button>See Project</button>
         </div>
-    </section>
-    <section class="project-container">
-        <div class="project">
-            <h2>Profesional Art Printing Data</h2>
+        <div class="project card4">
+            <h2>Profesional Art <br>Printing Data</h2>
             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
```

Comment:
- In your project container section, I'd like to kindly suggest that we follow the design pattern on the Figma template you chose. If you observe the template carefully, you'd notice that the first project card shows how each card should be implemented when hovered on. 

This means, the actual state appears to look a bit dark and stale, but when hovered on, it appears white with the `see project` button. Can we kindly implement this in your portfolio? This will help make your work easier by next week when implementing the popup modal. 

Here is what I am speaking about 👇🏽 

| The dark stale state of each card  |Should show this on hover |
| :---:   | :---: | 
| <img width="345" alt="Screenshot 2023-01-20 at 23 46 30" src="https://user-images.githubusercontent.com/99246359/213845595-6d0e0655-d62c-4001-9bba-10b749aa42ba.png"> |<img width="354" alt="Screenshot 2023-01-20 at 23 46 42" src="https://user-images.githubusercontent.com/99246359/213845670-2a8c8071-13b7-4026-a1c0-0e18e84a6ac3.png">  |

Can we kindly fix this? 
- All you need to do is to add some hover effect on your CSS indicating what should and should not show at certain times. 🌸

Reply:
Hi

Thank you,
I can make changes, but I can't find images. it is with the mask. Where can I get it?

Thanks